CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `Genesys` source to library.
 - Fixed a bug in `Databricks.create_table_from_pandas()`. The function that converts column names to snake_case was not used in every case. (#672)
 - Added `howto_migrate_sources_tasks_and_flows.md` document. This document will assist the DEs with the viadot 1 -> viadot 2 migration process.
-
+- Fixed a bug in `Databricks.create_table_from_pandas()`. DataFrame columns are now automatically converted using the `utils._cast_df_cols`. (#681)
```

Comment:
You missed out the key details, and now it's not clear neither what the exact issue was, nor what the solution is. You need to mention the `object` and `str` words somewhere in there.